art/art_dic.py

```diff
@@ -566,10 +566,17 @@
     "boombox2": "♫♪ |̲̅̅●̲̅̅|̲̅̅=̲̅̅|̲̅̅●̲̅̅| ♫♪",
     "equalizer": "▇ ▅ █ ▅ ▇ ▂ ▃ ▁ ▁ ▅ ▃ ▅ ▅ ▄ ▅ ▇",
     "playing cards": "[♥]]] [♦]]] [♣]]] [♠]]]",
+    "playing cards waterfall": "🂱🂲🂳🂴🂵🂶🂷🂸🂹🂺🂻🂼🂽🂾🃁🃂🃃🃄🃅🃆🃇🃈🃉🃊🃋🃌🃍🃎"
+                               "🃑🃒🃓🃔🃕🃖🃗🃘🃙🃚🃛🃜🃝🃞🂡🂢🂣🂤🂥🂦🂧🂨🂩🂪🂫🂬🂭🂮🂠🃏🃟",
+    "playing cards waterfall (trump)": "🃠🃡🃢🃣🃤🃥🃦🃧🃨🃩🃪🃫🃬🃭🃮🃯🃰🃱🃲🃳🃴🃵",
     "playing cards hearts": "[♥]]]",
+    "playing cards hearts waterfall": "🂠🂱🂲🂳🂴🂵🂶🂷🂸🂹🂺🂻🂼🂽🂾",
     "playing cards diamonds": "[♦]]]",
-    "playing cards spades": "[♠]]]",
+    "playing cards diamonds waterfall": "🃟🃁🃂🃃🃄🃅🃆🃇🃈🃉🃊🃋🃌🃍🃎",
     "playing cards clubs": "[♣]]]",
+    "playing cards clubs waterfall": "🃏🃑🃒🃓🃔🃕🃖🃗🃘🃙🃚🃛🃜🃝🃞",
+    "playing cards spades": "[♠]]]",
+    "playing cards spades waterfall": "🃠" + "🂡🂢🂣🂤🂥🂦🂧🂨🂩🂪🂫🂬🂭🂮",
```

Comment:
Here, two string are concatenated by `+` because of a weird manner.

Reply:
It might be a special case in vscode, it was alright with vim and other terminal-base editors.